daemon/cmd/endpoint.go

```diff
@@ -699,8 +699,9 @@ func (d *Daemon) deleteEndpointQuiet(ep *endpoint.Endpoint, conf endpoint.Delete
 }
 
 func (d *Daemon) DeleteEndpoint(id string) (int, error) {
-	// id can be an endpoint iD or an IP so not using logfields.EndpointID
-	log.WithField("id", id).Info("Delete endpoint request")
+	// The log message about request "DELETE /endpoint/{id} request" with level=debug has been
+	// in the previous step (deleteEndpointID Handle).
+	// So now needs a log message (level=info) with the information about EP to be deleted.
```

Comment:
I'm not sure I understand the need for these comments?

Reply:
I think too. I made them as an update.
So I've removed them.